tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -699,6 +700,15 @@ TEST (nnstreamer_nnfw_mlapi, invoke_single_02_n)
   ml_tensors_info_destroy (in_res);
 }
 
+#define wait_for_sink(expected_cnt) \
+  do { \
```

Comment:
And ```guint waiting_time = 0; \``` between LINE 704 and 705. And remove variable declaration of ```waiting_time``` from elsewhere.

Reply:
it's applied